fb-install-foreman.bats

```diff
@@ -161,7 +161,7 @@ EOF
 }
 
 @test "wake up puppet agent" {
-  puppet agent -t -v
```

Comment:
Well, -o --no-daemonize is a subset of the test options.. I think having the detailed error codes for this step is good, as we should throw an error on any change, as it's unexpected.